src/toil_scripts/adam_pipeline/spark_toil_script.py

```diff
@@ -0,0 +1,342 @@
+#!/usr/bin/env python2.7
+"""
+UCSC Computational Genomics Lab RNA-seq Pipeline
+Author: Audrey Musselman-Brown
+Affiliation: UC Santa Cruz Genomics Institute
+
+Please see the README.md in the same directory
+
+Toil pipeline for ADAM preprocessing
+
+                        Tree structure of ADAM pipeline
+                                       0 
+                                       |++(1)
+                                       2 --> 4 --> 5 --> 6 -> 7
+                                        ++(3)
+
+0 = Start Master
+1 = Master Service
+2 = Start Workers
+3 = Worker Service
+4 = Download Data
+5 = ADAM Convert
+6 = ADAM Transform
+7 = Upload Data
+
+================================================================================
+:Dependencies
+docker          - apt-get install docker (or 'docker.io' for linux)
+toil            - pip install --pre toil
+"""
+
+import argparse
+import os
+from subprocess import * #call, check_output
+import sys
+from toil.job import Job
+
+SPARK_MASTER_PORT = "7077"
+HDFS_MASTER_PORT = "8020"
+log = open("python.log", 'a')
+
+# - The tachyon and spark containers need the following ports open:
```

Comment:
I'll remove the comment. We've got the port mappings documented in cgl-docker-lib and are running with `--net=host` anyways.